src/Common/tests/System/PlatformDetection.cs

```diff
@@ -161,6 +161,7 @@ public static bool IsWinRT
         public static bool IsNotFedoraOrRedHatOrCentos => !IsDistroAndVersion("fedora") && !IsDistroAndVersion("rhel") && !IsDistroAndVersion("centos");
 
         public static bool IsFedora => IsDistroAndVersion("fedora");
+        public static bool IsRedHat69 => IsDistroAndVersion("rhel", "6.9") || IsDistroAndVersion("rhl", "6.9");
```

Comment:
I would check just the major version here, the minor one should not change the behavior, I believe.

Reply:
I would leave this as is for consistency with master - in this particular case we need to tell which version of ICU shipped with RedHat, I'm not sure what are the differences between 6.9 and other 6.X versions.

Reply:
There is no usable ICU version that ships with any version RedHat 6. We tell the users to install version 57.1 binary from the tarball provided on the ICU site. But even if there was one, RedHat never changes the major versions of libraries they ship when they bump the minor version. So 6.0, 6.1, ... 6.9 all would have the same major version of the library.

Reply:
@janvorli what @krwq meant is in 2.1 branch we detect the ICU version at runtime. so we don't care how the ICU get installed on the machine anyway. The right way to check the globalization behavior is to check the ICU version and not the OS version. We don't do that in v 2.0 servicing branches because this will need a change in the release bits (System.Globalzation.Native) and not the test bits only. This is why we are just doing the minimum needed in the servicing branch to have the tests not failing there.